File: 495-teemo-attacking/495-teemo-attacking.cs
public class Solution {
    public int FindPoisonedDuration(int[] time, int dur) {
        int sec =0;
        for(int i=0;i<time.Length-1;i++){
            if(time[i]+dur<=time[i+1])sec+=dur;
            else sec+=time[i+1]-time[i];
        }
        sec+=dur;
        return sec;
    }
}

File: 495-teemo-attacking/495-teemo-attacking.java
class Solution {
    public int findPoisonedDuration(int[] time, int dur) {
        int sec =0;
        for(int i=1;i<time.length;i++){
            sec+=Math.min(time[i]-time[i-1],dur); 
        }
        sec+=dur;
        return sec;
    }
}